api/openapi.yaml

```diff
@@ -1213,6 +1213,356 @@ paths:
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
 
+  # Notifications - Channels
+
+  /api/v1/notification/channels:
+    get:
+      operationId: listNotificationChannels
+      summary: List notification channels
+      description: |
+        List all notification channels.
```

Comment:
Adding an `example` would really help first time readers to get the concept quickly.